dockerfiles/mongodb-enterprise-appdb/10.2.15.5958-1_4.2.11-ent/ubuntu/Dockerfile

```diff
@@ -1,7 +1,9 @@
 ARG imagebase
 FROM ${imagebase} as base
 
-FROM ubuntu:xenial-20210416
+FROM ubuntu:xenial-20210114
```

Comment:
I will manually revert all these downgrades.

Reply:
I've created to avoid this in the future:

- https://github.com/10gen/ops-manager-kubernetes/pull/2194